lib/cfg/cfg.ts

```diff
@@ -56,7 +57,13 @@ export function generateStructure(compilerInfo: CompilerInfo, asmArr: AssemblyLi
     const isa = isLlvmIr ? 'llvm' : compilerInfo.instructionSet;
     const compilerGroup = isLlvmIr ? 'llvm' : isLLVMBased(compilerInfo) ? 'clang' : compilerInfo.group;
     const instructionSet = new (getInstructionSetByKey(isa ?? 'base'))();
-    const parser = new (getParserByKey(compilerGroup))(instructionSet);
+
+    // dex2oat is a special case because it can output different instruction
+    // sets. Create an OAT parser instead of searching by ISA.
+    const parser =
```

Comment:
It's unfortunate we need this here - but I couldn't think of a better way (unless @jeremy-rifkin has ideas)